README

```diff
@@ -138,3 +138,12 @@ build-history
                     │   ├── build-versions.inc
                     │   └── metadata-revs
                     └── prod-devel.xml
+
+Known issues
+============
+* If you use 'prod_pulls' parameter you may see message
+  "get_pulls() got an unexpected keyword argument 'base'".
+  This means that you have outdated PyGitHub on you PC.
+  PyGitHub v.1.45 is confirmed to works as expected.
```

Comment:
Imo, this means that we either need to put a `requirements.txt` or create `setup.py` to handle this
@lorc ?

Reply:
Latter, after submitting this patch, I found solution to remove dependency on PyGitHub. This solution is not yet submitted, and will be provided as separate pull request.
So I will drop this commit "Known issues" as not relevant.

Sorry that I didn't remove this commit earlier.